edk2toollib/uefi/telem/friendlynames.csv

```diff
@@ -0,0 +1,39 @@
+friendly name,guid
```

Comment:
why is this a CSV file? If it's for reference, it should be a markdown file so that it'll get published to the website. You can create a table in markdown I believe. If it's used in code, it should be converted to python so that it'll get included in the packaged pip module.